addon/mixins/taggable-mixin.js

```diff
@@ -8,7 +8,7 @@ export default Ember.Mixin.create({
             Ember.A(currentTags);
```

Comment:
It does not work without this line- it says that `currentTags.pushObject is not a function`.  I was using slice to make a copy and Ember.A to turn it into something I could do pushObject on.